### PR TITLE
chore: Django bump to <=5.0

### PR DESCRIPTION
**Pull request by xdaniel3**

### CODE REVIEW

File: auth_token/contrib/common/default/views.py [hunk 1]
```diff
@@ -60,11 +60,10 @@ class TokenLogoutView(LogoutView):
     @method_decorator(never_cache)
     def dispatch(self, request, *args, **kwargs):
         logout(request)
-        next_page = self.get_next_page()
-        if next_page:
+        if self.next_page:
```
Comment (Formulka) [inline auth_token/contrib/common/default/views.py]: `get_next_page` was renamed to `get_success_url` in django 4.1, shouldn't you use the `get_success_url` method?

**Review by rubickcz — APPROVED** [review]

**Review by HonzaSaibic — APPROVED** [review]

File: auth_token/contrib/common/default/views.py (hunk 1 above)
Comment (xdaniel3) [reply auth_token/contrib/common/default/views.py]: good catch :-) ten wrapper řeší aji redirecty

Comment (xdaniel3) [reply auth_token/contrib/common/default/views.py]: I removed it all together.
That logic was removed from dispatch method completely in Django 4.1. Compare https://github.com/django/django/blob/4.0.10/django/contrib/auth/views.py#L126 with https://github.com/django/django/blob/4.1.13/django/contrib/auth/views.py#L140

@Formulka please have a look

Comment (Formulka) [reply auth_token/contrib/common/default/views.py]: I see, thank you, yes this is ok 👍

File: setup.py [hunk 2]
```diff
@@ -24,13 +24,13 @@
         'Topic :: Internet :: WWW/HTTP',
     ],
     install_requires=[
-        'django>=2.2.14, <4.0',
+        'django>=4.2.0',
         'django-ipware>=3.0.2',
         'import_string==0.1.0',
-        'skip-django-chamber>=0.6.16.3',
-        'skip-django-generic-m2m-field>=0.0.9.3',
+        'skip-django-chamber @ git+https://github.com/skip-pay/django-chamber@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-chamber>=0.7.2',
```

File: setup.py [hunk 3]
```diff
@@ -24,13 +24,13 @@
...
+        'skip-django-generic-m2m-field @ git+https://github.com/skip-pay/django-generic-m2m-field@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-generic-m2m-field>=0.1.0',
```

File: setup.py [hunk 4]
```diff
@@ -24,13 +24,13 @@
...
         'skip-django-choice-enumfields>=1.1.3.2',
-        'skip-django-security-logger==1.6.4',
+        'skip-django-security-logger @ git+https://github.com/skip-pay/django-security@tda/chore/django_bump',
```
Comment (Formulka) [inline setup.py]:
```suggestion
        'skip-django-security-logger>=1.7.0',
```

File: example/requirements.txt [hunk 5]
```diff
@@ -1,9 +1,9 @@
-Django<=3.2
-skip-django-germanium==2.3.6.1
-skip-django-security-logger==1.6.4
+Django~=4.2.0
+skip-django-germanium @ git+https://github.com/skip-pay/germanium@tda/chore/django_bump
+skip-django-security-logger @ git+https://github.com/skip-pay/django-security@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements.txt]:
```suggestion
```

Comment (Formulka) [reply example/requirements.txt]: security is already in setup.py

File: example/requirements.txt [hunk 6]
```diff
@@ -1,9 +1,9 @@
-Django<=3.2
-skip-django-germanium==2.3.6.1
-skip-django-security-logger==1.6.4
+Django~=4.2.0
+skip-django-germanium @ git+https://github.com/skip-pay/germanium@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements.txt]:
```suggestion
skip-django-germanium==2.4.0
```

File: example/requirements.txt [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
+skip-django-security-logger @ git+https://github.com/skip-pay/django-security@tda/chore/django_bump
 coveralls
 flake8
-skip-django-is-core==2.24.6.1
-djangorestframework==3.12.2
+skip-django-is-core @ git+https://github.com/skip-pay/django-is-core@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements.txt]:
```suggestion
skip-django-is-core==2.25.0
```

**Review by Formulka — APPROVED** [review]

**Pull request merged**